dev/cloud/teardown.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Magento\Framework\App\Bootstrap;
+require '/app/app/bootstrap.php';
+
+$bootstrap = Bootstrap::create(BP, $_SERVER);
+$obj = $bootstrap->getObjectManager();
+
+$algoliaHelper = $obj->get('\Algolia\AlgoliaSearch\Helper\AlgoliaHelper');
+
+/**
+ * @param $algoliaHelper Algolia\AlgoliaSearch\Helper\AlgoliaHelper
+ * @param array $indices
+ */
+function deleteIndexes($algoliaHelper, array $indices)
```

Comment:
```suggestion
function deleteIndexes($algoliaHelper, array $indices, $magentoCloudIndexPrefix)
```

Reply:
If a function uses a `getenv`, it means it has a [tight coupling](https://williamdurand.fr/2013/07/30/from-stupid-to-solid-code/#tight-coupling) with way the script is launched. I believe it's good practice to avoid that. Trying to apply the same principles in every snippet of PHP would allow us to copy/move code from scripts into our main code base without worrying.